packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -122,3 +126,14 @@ function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>
     </Flex>
   );
 }
+
+function Example() {
+  let [show, setShow] = React.useState(false);
+  let [show2, setShow2] = React.useState(false);
+  return (
+    <>
+      <Button variant="cta" UNSAFE_style={show && {background: 'red', userSelect: 'text'}} onPressStart={() => setTimeout(() => setShow(true), 3000)}>Press and hold (overwrite)</Button>
+      <Button variant="cta" UNSAFE_style={show2 && {background: 'red'}} onPressStart={() => setTimeout(() => setShow2(true), 3000)}>Press and hold (no overwrite)</Button>
+    </>
+  );
+}
```

Comment:
Brought back my previous approach since returning the `user-select` styles from usePress caused issues when the user had inline styles applied. There was a concern brought up previously about what would happen if the styles of an element were changed during press so I made these examples to illustrate what would happen. Press and hold these button to see what happens to the inline styles. If we deem the current approach acceptable, I'll spruce up these examples